flow/connectors/bigquery/qrep_avro_sync.go

```diff
@@ -184,18 +190,18 @@ func (s *QRepAvroSyncMethod) SyncQRepRecords(
 	)
 	bqClient := s.connector.client
 
+	insertColumns := make([]string, 0, len(dstTableMetadata.Schema))
+	for _, col := range dstTableMetadata.Schema {
+		insertColumns = append(insertColumns, fmt.Sprintf("`%s`", col.Name))
+	}
```

Comment:
Need to include synced_at/deleted columns

Reply:
for example that's how we're setting their values in getTransformedColumns